cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -431,7 +431,9 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             if (!this.isValidTimeLimit(this.$('.field-time-limit input').val())) {
                 this.$('.field-time-limit input').val('00:30');
             }
-            if (showRulesField) {
+
+            let  showReviewBlock = this.model.get('show_review_rules');
```

Comment:
There are two spaces between `let` and `showReviewBlock`, and there should be one. I suspect the eslint check is failing because of this.

Reply:
Oh crud we also shouldn't use `let`; we need to write ES5 here to support IE 11. I would not expect this JS to run through babel

Reply:
no longer relevant.